src/main/java/duke/task/Task.java

```diff
@@ -148,4 +148,9 @@ public String getDateString() {
         return null;
     }
 
+    public boolean contains(String arr1) {
```

Comment:
I think this should have an "is" in front

Consider refactoring to isInside, isContained etc.

Reply:
Good catch, I shall change it now.

Reply:
Thank you for the changes! :)